README.md

```diff
@@ -20,3 +20,31 @@ app/bin/app --help
 # On Windows:
 app/bin/app.bat --help
 ```
+
+## Examples
+
+These examples use gradle to simplify testing.
+
+### Set single value
+
+```shell
+./gradlew app:run --args="-x identifier -w -s 42 -t 'consignment/deliveryEvent/actualOccurrenceDateTime:=123456789012+0000'"
```

Comment:
Should the example have a valid datetime instead?

Reply:
Good point, fixed.